prometheus_client/values.py

```diff
@@ -12,16 +13,23 @@ class MutexValue:
 
     def __init__(self, typ, metric_name, name, labelnames, labelvalues, help_text, **kwargs):
         self._value = 0.0
+        self._timestamp_sec = 0.0
```

Comment:
I think I would slightly prefer just `timestamp` as the variable name across this PR. I don't think we have units in other variable names at least.